exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
+    "cases": [
+      {
+        "description": "transforms one value",
```

Comment:
Thanks @Insti for all these.  I think i'm getting it.  I'll make updates accordingly